drivers/hmis/app/graphql/mutations/submit_assessment.rb

```diff
@@ -54,6 +54,9 @@ def resolve(input:)
       assessment_detail_valid = assessment.assessment_detail.valid?
 
       if assessment_valid && assessment_detail_valid
+        # call manually to save enrollment. figure out a better way
+        assessment.assessment_detail.assessment_processor.save!
```

Comment:
@eanders  here is my hacky solution:
This line, along with the `before_save :save_enrollment` hook in the processor, allows the Enrollment to save. I would love to figure out a way for the enrollment to save magically when the assessment_detail saves, like the rest of the records do.